move/its/sources/its.move

```diff
@@ -255,6 +261,8 @@ public fun burn_as_distributor<T>(
     );
 }
 
+// This is the entrypoint for operators to set the flow limits of their tokens
+// (tokenManager.setFlowLimit on EVM)
```

Comment:
```suggestion
/// Set the flow limit for a given tokenId
```
Use `///` for docstrings. Also mention how to remove the flow limit.

Reply:
done